.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: Release Images
+
+on:
+  workflow_dispatch:
+    inputs:
+      commit:
+        description: "Commit sha"
+        required: true
+      tag:
+        description: "The tag for the new images"
+        required: true
+
+jobs:
+  retag-and-release:
+    name: "Re-tag and release images"
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+      - name: Set up skopeo
+        run: |
+          skopeo() {
+            # Needed for older docker versions
+            # See: https://github.com/containers/skopeo/issues/1501 and https://github.com/moby/moby/issues/42680
+            local seccomp="--security-opt seccomp=unconfined"
+
+            docker run --rm -i $seccomp quay.io/skopeo/stable "$@"
+          }
+          skopeo login -u ${{ secrets.DOCKER_USERNAME }} --password ${{ secrets.DOCKER_PASSWORD }} docker.io
```

Comment:
ideally, use env vars for these so you can do the quoting properly.

(but at the very least, put the substitutions in single quotes)